tasks.yaml

```diff
@@ -110,6 +110,14 @@ tasks:
     actions:
       - task: test:uds-core-upgrade
 
+  - name: test-compliance-validate
+    actions:
+      - task: test:compliance-validate
+
+  - name: test-compliance-evaluate
+    actions:
+      - task: test:compliance-evaluate
+
```

Comment:
Can we add descriptions of these commands at this level?

Reply:
for sake of clarifying intent - are you requesting the `description` field be added to these?  

IE 
```yaml
- name: test-compliance-evaluate
  description: Evaluation of compliance given an established threshold
  actions:
    - task: test:compliance-evaluate
```

or something else?

Reply:
Yep.

Reply:
Added